bin/lookup_evolve.py

```diff
@@ -104,7 +104,7 @@ def mutate(self):
 
     @staticmethod
     def crossover_tables(table1, table2):
-        keys = list(sorted(table1.keys()))
+        keys = list(table1.keys())
```

Comment:
I think we still want the sort here, or is there a good reason to remove it?

Reply:
I removed it because it was throwing this error:

`TypeError: '<' not supported between instances of 'Action' and 'Action'` 

I considered using a comparator or another method to sort, but I don't think sorting is actually needed here. The keys are considered in the same order for both table1 and table2, so the crosspoint still defines the same partition of the keys for both tables, which seems sufficient.